_data/xcodes.yml

```diff
@@ -1,4 +1,43 @@
 osx_images:
+  - image: xcode9
+    xcode: "9.0"
+    osx_version: "10.12"
+    xcode_full_version: "9 beta 2"
```

Comment:
I'd prefer a capital B just to be in line with how it looks for everything else, but it's not super important. :b: